### PR TITLE
Make make-edgelist.c consistent, fixing whitespace, adding graph500 binary format to edgelist converter.

### PR DESCRIPTION
**Pull request by sampollard**
Everything except the changes to make-edgelist.c are just quality of life improvements.

graph5002el is a very braindead converter but it works as long as the graph can fit in memory.

### CODE REVIEW

**Comment by marcinz** [comment]
Sam, 

Would it be possible for you to split this merge request into smaller requests, each with one feature?

Here are some comments for now.

What is the use case for the converter? I can see how it is useful in general, but why is it useful in the graph500 repo? Is there any scenario that would require its use?

I think it would be nice if by default the output matches the required fields in http://www.graph500.org/specifications#sec-8_3 . Perhaps additional output could be enabled by a flag?

I see that changes to make-edgelist.c contains many whitespace changes. Could you separate these from the feature changes? Also, could you summarize the changes to make-edgelist.c and compare them with the merge request I have merged in today?

Thank you.

**Comment by sampollard** [comment]
Marcin,

Thank you for the feedback. I will cancel this pull request and submit smaller ones. I think the converter and the additional output are both too specific to what I am trying to do so I will not include them in future pull requests.

I do have one additional issue: when trying to load a graph using the -o and -r flags in the following commands:

```
./make-edgelist -s 13 -o kron13.graph500 -r kron13.roots
seq-csr/seq-csr -s 13 -o kron13.graph500 -r kron13.roots
```

I get the following error:

```bfs 0 from 26 failed verification (-999)```

It occurs in both the seq-csr and omp-csr versions. Of course, it is not a critical issue since it is faster to just regenerate the data for each run, but I figured it was worth knowing about.

**Comment by marcinz** [comment]
Hmm, verification failing is a bad sign. It is possibly a bug of some sort. If you could look into why verification fails, that would be very helpful. Perhaps there is something wrong with the input data? Does the first root on your list have any neighbors?

**Comment by sampollard** [comment]
I think I fixed the issue. I made a new pull request. I apologize I am not very familiar with git branching yet so I hope I did everything right.
